pkg/infrastructure/services/manage_status.go

```diff
@@ -44,8 +40,9 @@ func ManageStatus(instance *v1alpha1.KogitoJobsService, cli *client.Client, errC
 
 	updateStatus := false
 	changed := false
-	updateStatus = updateImageStatus(instance) || updateStatus
-	if changed, err = updateDeploymentStatus(instance, cli); err != nil {
+	updateStatus = updateImageStatus(instance, imageName) || updateStatus
+	var readyReplicas int32
```

Comment:
What did you mean by `envs`? This `readyReplicas` var is to provide a feedback for the user that the service is already provisioned.

Reply:
yeah, but, cant be it assigned on the if statement?

Reply:
It can't since the `if statement has its own scope and we will need that var later in the function, that's why it's only declared there. Plus, we need to specify that the var is `int32` because of the next comparisons. I've changed the `updateStatus` var to be declared and assigned in the same line, though. This is a copy and paste from other function, I missed that. Thanks! :)